### ISSUE TITLE
Sporadically changing the orientation of uploaded image.

### ISSUE THREAD

**Issue by hope-portal-services** [likeconan/react-avatar-image-cropper#28]
The uploaded image will be a portrait style photo, when the preview renders the image, the image is in landscape. Happens sporadically across different devices.

**Comment by likeconan** [comment]
Good catch! Could you please provide what devices you occur this issue?

**Comment by hope-portal-services** [comment]
I have seen this on iPhone 8, iPad, and a Windows 10 Laptop. It's been very difficult to duplicate or pinpoint when/why this happens so I'm unable to provide much diagnostic information. The only pattern I notice is that when this does happen, the images are usually close to 2MB in size, so larger scale images, and most often on Apple devices.

**Comment by te-online** [comment]
@likeconan It looks like most browsers can figure out the rotation by themselves. The `resetOrientation` method then reverts the correction. Is that possible? 

If you take this [example file](https://github.com/recurser/exif-orientation-examples/blob/master/Portrait_3.jpg): The `resetOrientation` method reads an orientation `3` and rotates accordingly. The result, however, is that the image is on its head. With `resetOrientation` disabled, the image is rotated correctly. 

I tested in Firefox 88.0.1 and Safari 14.1 on macOS.

I'm wondering if there's a way to tell if the browser already rotated the image according to EXIF, so we can avoid applying the `resetOrientation` then? 🤔

**Comment by likeconan** [comment]
> @likeconan It looks like most browsers can figure out the rotation by themselves. The `resetOrientation` method then reverts the correction. Is that possible?
> 
> If you take this [example file](https://github.com/recurser/exif-orientation-examples/blob/master/Portrait_3.jpg): The `resetOrientation` method reads an orientation `3` and rotates accordingly. The result, however, is that the image is on its head. With `resetOrientation` disabled, the image is rotated correctly.
> 
> I tested in Firefox 88.0.1 and Safari 14.1 on macOS.
> 
> I'm wondering if there's a way to tell if the browser already rotated the image according to EXIF, so we can avoid applying the `resetOrientation` then? 🤔

That's a great suggestion. Thanks a lot and let me try to research and verify
